src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populationInfo = document.querySelectorAll('.population');
+
+const arrOfStatistics = [...populationInfo]
```

Comment:
Consider using more descriptive variable names, like 'countryPopulations' instead of 'arrOfStatistics'.